test/smartwallet/CustomSmartWallet.test.ts

```diff
@@ -127,8 +133,8 @@ describe('Custom Smart Wallet using TestToken', function () {
     failureCustomLogic = await failureCustomLogicFactory.deploy();
     proxyCustomLogic = await proxyCustomLogicFactory.deploy();
 
-    const provider = ethers.provider;
-    owner = ethers.Wallet.createRandom().connect(provider);
+    const provider = new providers.JsonRpcProvider(RSK_URL);
+    owner = Wallet.createRandom().connect(provider);
 
     [relayHub, worker, fundedAccount] = (await ethers.getSigners()) as [
```

Comment:
```suggestion
     [relayHub, worker, fundedAccount] = await ethers.getSigners();
```

Reply:
It was part of the already merged PR PP640. I tried your suggestion but it makes the code fail.